rolling/distribution.yaml

```diff
@@ -3721,11 +3721,12 @@ repositories:
       - mrpt_pointcloud_pipeline
       - mrpt_rawlog
       - mrpt_reactivenav2d
+      - mrpt_tps_astar_planner
```

Comment:
- [x] Top level license file:
- [x] License is [OSI-approved](https://opensource.org/licenses):
- [x]  License correctly listed in package.xmls
- [x]  Public source repo:
- [x]  Source repository contains ROS packages
- [x]  Each package meets [REP-144](https://www.ros.org/reps/rep-0144.html) naming conventions